libs/templates/src/lib/home/sections/hero/data.tsx

```diff
@@ -72,8 +72,12 @@ export const renderHomeHeroSliderContent = () => (
 export const HomeHeroForm = () => {
   return (
     <div className={'flex flex-col gap-general-lg'}>
-      <div className="flex flex-col items-center justify-between gap-gap-lg md:flex-row">
-        <TextField placeholder="Email Address" type="email" />
+      <div className="flex flex-col items-center justify-between gap-gap-md md:flex-row">
+        <TextField
+          placeholder="Email Address"
+          type="email"
+          className="w-full"
```

Comment:
do we need this `className` if it's getting fixed in quill?

Reply:
will remove this `classname`